src/sentry/search/snuba/executors.py

```diff
@@ -1141,7 +1143,24 @@ class GroupAttributesPostgresSnubaQueryExecutor(PostgresSnubaQueryExecutor):
             ),
             0,
         ],
+        alias="score",
     )
+    first_seen = Column("group_first_seen", entities["attrs"])
+
+    sort_strategies = {
+        "date": "last_seen",
+        "new": "first_seen",
+    }
+
+    sort_defs = {
+        "first_seen": first_seen,
+        "last_seen": last_seen_aggregation,
+    }
+
+    column_sort_mapping = {
+        "new": "g.group_first_seen",
+        "date": "score",
+    }
```

Comment:
I don't think we need to tackle this in this pr, but this feels like potentially these should be one data structure, like:

```
sort_strategies = {
    "new": SortStrategy(
         snql_definition=Column("group_first_seen", entities["attrs"]), 
         result_name="g.group_first_seen",
         ...
    ),
    "date": ...
}
```

Might just be a little easier to work with instead of having to look up dictionaries, and remember to add fields to each dictionary